builtin/providers/aws/resource_aws_sns_topic.go

```diff
@@ -183,9 +185,14 @@ func resourceAwsSnsTopicRead(d *schema.ResourceData, meta interface{}) error {
 				// Some of the fetched attributes are stateful properties such as
 				// the number of subscriptions, the owner, etc. skip those
 				if resource.Schema[iKey] != nil {
-					value := *attrmap[oKey]
+					var value string
+					if iKey == "policy" {
```

Comment:
Ops, sorry. It should not be the same fixed. I don't want to change the behaviour of the other attributes, that is why I got the if.